### PR TITLE
prim::max transformation for ListConstruct

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: src/frontends/pytorch/src/utils.cpp [hunk 1]
```diff
@@ -118,6 +118,22 @@ std::shared_ptr<Node> numel(NodeContext& context, size_t input_id) {
     return context.mark_node(std::make_shared<opset8::ReduceProd>(input_shape, axes, false));
 };
 
+
```
Comment (mvafin) [inline src/frontends/pytorch/src/utils.cpp]: Please fix codestyle

File: src/frontends/pytorch/src/utils.cpp [hunk 2]
```diff
@@ -118,6 +118,21 @@ std::shared_ptr<Node> numel(NodeContext& context, size_t input_id) {
     return context.mark_node(std::make_shared<opset8::ReduceProd>(input_shape, axes, false));
 };
 
+std::shared_ptr<Node> concat_list_construct(std::shared_ptr<Node> input) {
+    if (auto list_construct = cast_fw_node(input, "prim::ListConstruct")) {
+        auto list_inputs = list_construct->input_values();
+        auto node_vector = NodeVector();
```
Comment (mvafin) [inline src/frontends/pytorch/src/utils.cpp]:
```suggestion
        OutputVector node_vector;
```

Comment (mvafin) [reply src/frontends/pytorch/src/utils.cpp]: Better to use `OutputVector`.

File: src/frontends/pytorch/src/utils.cpp [hunk 3]
```diff
@@ -118,6 +118,21 @@ std::shared_ptr<Node> numel(NodeContext& context, size_t input_id) {
     return context.mark_node(std::make_shared<opset8::ReduceProd>(input_shape, axes, false));
 };
 
+std::shared_ptr<Node> concat_list_construct(std::shared_ptr<Node> input) {
```
Comment (mvafin) [inline src/frontends/pytorch/src/utils.cpp:121]: Do you plan to use this function anywhere except in `MaxPrimListConstructReplacer`? If no I would recommend moving this function to the file with transformation.

**Review by mvafin — APPROVED** [review]

File: src/frontends/pytorch/src/utils.cpp (hunk 3 above)
Comment (eaidova) [reply src/frontends/pytorch/src/utils.cpp:121]: yes, I plan to reuse it, I think similar logic for other ops where ListConstruct can be used as shapes (reshape, view, pad) should be used

**Pull request merged**